src/main/resources/mixins.common.realtime.json

```diff
@@ -0,0 +1,28 @@
+{
+  "minVersion": "0.5.1",
+  "package": "org.spongepowered.common.mixin.realtime.mixin",
```

Comment:
@liach 

His interface is the same name as the one in common and adds new things. His location of it is fine and won't be moved.

@Aaron1011 
You oughta know by this point that whatever is in the root or child of root mixin page isn't classloaded and therefore cannot be referenced in managed code.